src/config.rs

```diff
@@ -16,6 +16,7 @@ const DEFAULT_FONT_COLOR: Color = Color::from_rgba(0xf8, 0xf8, 0xf2, 0xff);
 const DEFAULT_BG_COLOR: Color = Color::from_rgba(0x27, 0x28, 0x22, 0xee);
 const DEFAULT_INPUT_BG_COLOR: Color = Color::from_rgba(0x75, 0x71, 0x5e, 0xc0);
 const DEFAULT_SELECTED_FONT_COLOR: Color = Color::from_rgba(0xa6, 0xe2, 0x2e, 0xff);
+const DEFAULT_BG_BORDER_COLOR: Color = Color::from_rgba(0xfe, 0x11, 0x11, 0xff);
```

Comment:
```suggestion
const DEFAULT_BG_BORDER_COLOR: Color = Color::from_rgba(0x13, 0x14, 0x11, 0xff);
```

It's fine for a test but too contrasting with the DEFAULT_BG_COLOR